packages/gasket-plugin-elastic-apm/README.md

```diff
@@ -28,35 +28,35 @@ module.exports = {
 }
 ```
 
-## Configuration
-
-Configurations for the plugin can be added under `elasticAPM` in the config.
-This object accepts the same properties as the Elastic APM Node.js agent. (See
-the [configuration options documentation])
+Add a `--require` flag to a `package.json` start script:
 
-#### Example configuration
-
-```js
-module.exports = {
-  plugins: {
-    add: ['@gasket/plugin-elastic-apm']
-  },
-  elasticAPM: {
-    secretToken: '****',
-    serverUrl: 'http://localhost:9200'
+```diff
+  "scripts": {
+    "build": "gasket build",
+-   "start": "gasket start",
++   "start": "gasket start --require elastic-apm-node/start",
```

Comment:
I'm a little concerned that this completely bypasses the plugin and the Gasket config system, because it diminishes our ability to further modify the agent's configuration when it starts. Could the plugin itself expose a module that can be used with the `--require` syntax? It could then read configuration data from either `gasket.config.js` or environment variables and start the agent.

```suggestion
+   "start": "gasket start --require @gasket/plugin-elastic-apm/start",
```

Also, what happens if the upstream repo moves or renames `elastic-apm-node/start`? By not directly requiring a file within an external package, we can provide a better guarantee that this startup method will continue to work if the upstream changes significantly.

Reply:
Yes, unfortunately it does bypass the Gasket engine. But, if you want to ensure the agent bootstraps the many modules brought in from loading plugins to set up the Gasket engine, then it has to start before it.

This scaffolds the minimum configuration to get started. `elastic-apm-node/start` is the recommended API given by the elastic docs, so I wouldn't expect it to move. Still, app teams could choose wrap it and do additional configuration in a custom setup script.

```bash
gasket start --require ./setup.js
```
```js
// setup.js
module.exports = {
  require('elastic-apm-node').start({
    // whatever you want
  })
}
```

Reply:
I can add this as a note in the readme if you think that'd help ☝️

Reply:
looks great!